.changeset/fast-poems-beg.md

```diff
@@ -0,0 +1,66 @@
+---
+"@turnkey/ethers": minor
+---
+
+Updates @turnkey/ethers package and examples to use ethers v6. Refer to https://docs.ethers.org/v6/migrating for full migration instructions.
+
+✨Summary of Changes✨
+
+- `getBalance` iis no longer a method on the signer. It must be obtained via the provider instance.
```

Comment:
because each of these changes comes with a code snippet, i think it's fine formatting-wise if we don't use bullet points at all! but very subjective here :)